internal/featuresupport/features_olm_operators.go

```diff
@@ -164,6 +164,10 @@ func (feature *CnvFeature) getSupportLevel(filters SupportLevelFilters) models.S
 		return models.SupportLevelUnavailable
 	}
 
+	if filters.PlatformType != nil && (*filters.PlatformType == models.PlatformTypeNutanix || *filters.PlatformType == models.PlatformTypeVsphere) {
```

Comment:
can you add a test case for
`filters.PlatformType` not defined? 

I

Reply:
Added